models/algorithms/calibratedClassifier.py

```diff
@@ -181,8 +181,7 @@ def fit(self, X, y, sample_weight=None, eval_set=None, sample_weight_eval_set=No
             import ml_insights as mli
             self.calib_method = "spline"
             spline = mli.SplineCalib(
-                penalty='l2',
-                solver='liblinear',
+                method="L-BFGS-B",
```

Comment:
@IzuiT Previously `ml_insights` was using `sklearn.linear_model.LogisticRegression` for spline fitting. So `penalty` and `solver` parameters came there (see https://github.com/numeristical/introspective/commit/9d4f5be0b159269bf61d66a0364b151f9aec5584#diff-6298dc7bfb7d0111ff50669c3514cc651dbfdc201be31e81d1d697df1db9df8fL23-L34). Now `ml_insights` uses (via `splinecalib`) `scipy.optimize.minimize` instead (see https://github.com/numeristical/splinecalib/commit/45dc522c000c8e90243ce7d5024bc26b2f211f36#diff-b39b488cbeef80f1d0137a05757ab135c3037ea3129c140eb108b26458c89625R23-R25). Therefore they have replaced the `penalty` and `solver` param with the `method` param.

I'm having a hard time mapping the `penalty='l2'` & `solver='liblinear'` to the new `method` param. I couldn't find the direct replacement in scipy doc. For your reference [`sklearn.linear_model.LogisticRegression.html` docs](https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LogisticRegression.html#sklearn.linear_model.LogisticRegression), [`scipy.optimize.minimize.html` docs](https://docs.scipy.org/doc/scipy/reference/generated/scipy.optimize.minimize.html#scipy.optimize.minimize)

Reply:
With `method == "L-BFGS-B"` you've made a right choice. You also might consider using `method == "BFGS"` for small to mid-sized datasets (up to 1M samples) but I would stick to `L-BFGS-B` only.